e2e/provision/README.md

```diff
@@ -66,6 +73,71 @@ gcloud compute ssh ubuntu@nephio-r1-e2e -- \
                 sudo journalctl -u google-startup-scripts.service --follow
 ```
 
+## Installing on a pre-provisioned VM
+
+This install has been verified on VMs running on Openstack, AWS, and Azure. 
+
+### VM Prerequisites
+
+Order or create a VM with the following specification:
+
+- Linux Flavour: Ubuntu-20.04-focal
+- 8 cores
+- 32 GB memory
+- 200 GB disk size
+- default user with sudo passwordless permissions
+
+**Configure a route for Kubernetes**
+
+In some installations, the IP range used by Kubernetes in the sandbox can clash with the
+IP address used by your VPN. In such cases, the VM will become unreachable during the
+sandbox installation. If you have this situation, add the route below on your VM.
+
+Log onto your VM and run the following commands,
+replacing **\<interface-name\>** and **\<interface-gateway-ip\>** with your VMs values: 
+
+```bash
+sudo bash -c 'cat << EOF > /etc/netplan/99-cloud-init-network.yaml
+network:
+  ethernets:
+    <interface-name>:
+      routes:
+        - to: 172.18.2.6/32
+          via: <interface-gateway-ip>
+          metric: 100
+  version: 2
+EOF'
+
+sudo netplan apply
+```
+
+### Kick off installation on VM
+
+Log onto your VM and run the following command:
+
+```bash
+wget -O - https://raw.githubusercontent.com/nephio-project/test-infra/main/e2e/provision/init.sh |  \
+sudo NEPHIO_DEBUG=false   \
+     NEPHIO_USER=ubuntu  \
+     bash
+```
+
+The following environment variables can be used to configure the installation:
+
+| Variable               | Values           | Default Value | Description                                            |
+| ---------------------- | ---------------- | ------------- | ------------------------------------------------------ |
+| NEPHIO_USER            | userid           | ubuntu        | The user to install the sandbox on (must have sudo passwordless permissions) |
+| NEPHIO_DEBUG           | false or true    | false         | Controls debug output from the install                 |
+| NEPHIO_DEPLOYMENT_TYPE | r1 or one-summit | r1            | Controls the type of installation to be carried out    |
+| RUN_E2E                | false or true    | false         | false: install management cluster only<br>true: install management and  workload clusters |
+| NEPHIO_REPO            | URL              | https://github.com/nephio-project/test-infra.git |URL of repo to use for installation |
```

Comment:
```suggestion
| RUN_E2E                | false or true    | false         | Determines whether the End-to-End tests are going to be executed |
| NEPHIO_REPO            | URL              | https://github.com/nephio-project/test-infra.git | URL of the repository to be used for installation |
```

Reply:
Done, thanks for correcting the RUN_E2E description, I misunderstood that flag.